[2023-07-30 22:05:25,419] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: dag_smtp_email_automatico.dag_envio scheduled__2023-03-25T00:00:00+00:00 [queued]>
[2023-07-30 22:05:25,564] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: dag_smtp_email_automatico.dag_envio scheduled__2023-03-25T00:00:00+00:00 [queued]>
[2023-07-30 22:05:25,565] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-07-30 22:05:25,565] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2023-07-30 22:05:25,565] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-07-30 22:05:25,734] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): dag_envio> on 2023-03-25 00:00:00+00:00
[2023-07-30 22:05:25,755] {standard_task_runner.py:52} INFO - Started process 1282 to run task
[2023-07-30 22:05:25,788] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dag_smtp_email_automatico', 'dag_envio', 'scheduled__2023-03-25T00:00:00+00:00', '--job-id', '201', '--raw', '--subdir', 'DAGS_FOLDER/dag_smtp_email.py', '--cfg-path', '/tmp/tmpzpq38sv7', '--error-file', '/tmp/tmpq0g3xx5h']
[2023-07-30 22:05:25,789] {standard_task_runner.py:80} INFO - Job 201: Subtask dag_envio
[2023-07-30 22:05:26,673] {task_command.py:371} INFO - Running <TaskInstance: dag_smtp_email_automatico.dag_envio scheduled__2023-03-25T00:00:00+00:00 [running]> on host b0e2d990d7ae
[2023-07-30 22:05:27,203] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Ariel Feldman
AIRFLOW_CTX_DAG_ID=dag_smtp_email_automatico
AIRFLOW_CTX_TASK_ID=dag_envio
AIRFLOW_CTX_EXECUTION_DATE=2023-03-25T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-03-25T00:00:00+00:00
[2023-07-30 22:05:28,974] {logging_mixin.py:115} INFO - (535, b'5.7.8 Username and Password not accepted. Learn more at\n5.7.8  https://support.google.com/mail/?p=BadCredentials i17-20020a9d6251000000b006b9f26b9b94sm3614113otk.28 - gsmtp')
[2023-07-30 22:05:28,975] {logging_mixin.py:115} INFO - Failure
[2023-07-30 22:05:28,975] {python.py:173} INFO - Done. Returned value was: None
[2023-07-30 22:05:29,120] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=dag_smtp_email_automatico, task_id=dag_envio, execution_date=20230325T000000, start_date=20230730T220525, end_date=20230730T220529
[2023-07-30 22:05:29,269] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-07-30 22:05:29,425] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
